app/views/articles/show.html.erb

```diff
@@ -115,7 +115,7 @@
               <% if @article.search_optimized_title_preamble.present? && !user_signed_in? %>
                 <span class="fs-xl color-base-70 block"><%= @article.search_optimized_title_preamble %></span>
               <% end %>
-              <%= sanitize_and_decode @article.title %>
+              <%= @article.title %>
```

Comment:
The reason that this is the solution is that Rails' ERB template is already sufficient at rendering given text as is, ie. no processing and already escaped. `sanitize_and_decode` was originally implemented to try to fix escaped text on the feed, which is actually handled via JS. It wasn't needed here.